internal/report/report.go

```diff
@@ -98,7 +120,27 @@ func (r *PolicyReport) AddResult(policy policiesv1.Policy, resource unstructured
 		).Msg("added result to report")
 }
 
-func (r *ClusterPolicyReport) AddResult(policy policiesv1.Policy, resource unstructured.Unstructured, auditResponse *admv1.AdmissionReview, responseErr error) {
+// GetSummaryJSON gets the report.Summary formatted in JSON. Useful for logging
+func (r *PolicyReport) GetSummaryJSON() (string, error) {
+	marshaled, err := json.Marshal(r.Summary)
+	if err != nil {
+		return "error marshalling summary", err
+	}
+	return string(marshaled), nil
+}
+
+func (r *ClusterPolicyReport) GetSummary() (string, error) {
```

Comment:
Shouldn't we rename this to `GetSummaryJSON` as well?

Reply:
indeed! done in https://github.com/kubewarden/audit-scanner/pull/47/commits/8421778c7678549954128966d91d3308a28a7422.